OSDOCS-13041: Documented the 4.14.44 z-stream RNs (#86727)
openshift/openshift-docs: release_notes/ocp-4-14-release-notes.adoc

{product-title} release 4.14.44, which includes security updates, is now available. The list of bug fixes that are included in the update is documented in the link:https://access.redhat.com/errata/RHSA-2025:0029[RHSA-2025:0029] advisory. The RPM packages that are included in the update are provided by the link:https://access.redhat.com/errata/RHBA-2025:0032[RHBA-2025:0032] advisory.

Space precluded documenting all of the container images for this release in the advisory.

You can view the container images in this release by running the following command:

[source,terminal]
----
$ oc adm release info 4.14.44 --pullspecs
----

[id="ocp-4-14-44-bug-fixes_{context}"]
==== Bug fixes

* Previously, the Single-Root I/O Virtualization (SR-IOV) Operator did not expire the acquired lease during the Operator's shutdown operation. This impacted a new instance of the Operator, because the new instance had to wait for the lease to expire before the new instance was operational. With this release, an update to the Operator shutdown logic ensures that the Operator expires the lease when the Operator is shutting down. (link:https://issues.redhat.com/browse/OCPBUGS-44726[*OCPBUGS-44726*])

> * Previously, when you attempted to use {olm-first} to upgrade an Operator, the upgrade was blocked and an `error validating existing CRs against new CRD's schema` message was generated. An issue existed with {olm}, whereby {olm} erroneously identified incompatibility issues validating existing custom resources (CRs) against the new Operator version's custom resource definitions (CRDs). With this release, the validation is corrected so that Operator upgrades are no longer blocked. (link:https://issues.redhat.com/browse/OCPBUGS-46595[*OCPBUGS-46595*]) 

Author: Well spotted. The attributes do not exist in the common-attributes.adoc file for 4.14 but they do exist for other releases.

> * Previously, the `aws-sdk-go-v2` software development kit (SDK) failed to authenticate an `AssumeRoleWithWebIdentity` API operation on an {aws-first} {sts-first} cluster. With this release, `pod-identity-webhook` now includes a default region, and this issue no longer persists. (link:https://issues.redhat.com/browse/OCPBUGS-46487[*OCPBUGS-46487*])

Reviewer: 1. Suggestion to double-check if `now` is needed here, considering that the sentence already starts with `With this release`: https://redhat-documentation.github.io/supplementary-style-guide/#now.
2. Just wondering if `persists` is the best word choice here (could using `persists` be understood in the sense that this issue still appears but doesn't linger?).
```suggestion
* Previously, the `aws-sdk-go-v2` software development kit (SDK) failed to authenticate an `AssumeRoleWithWebIdentity` API operation on an {aws-first} {sts-first} cluster. With this release, `pod-identity-webhook` includes a default region, which prevents such authentication failures. (link:https://issues.redhat.com/browse/OCPBUGS-46487[*OCPBUGS-46487*])
```

> * Previously, when you used the Agent-based Installer to install a cluster on a node that had an incorrect date, the cluster installation failed. With this release, a patch is applied to the Agent-based Installer live ISO time synchronization. The patch fixes the date issue and configures the `/etc/chrony.conf` file with the list of additional Network Time Protocol (NTP) servers, so that you can set any of these additional NTP servers in the `agent-config.yaml` without experiencing a cluster installation issue. (link:https://issues.redhat.com/browse/OCPBUGS-45464[*OCPBUGS-45464*])

Reviewer: 1. Just wondering if the developer has this meaning in mind.
2. The last sentence could be split, and the last clause moved to another sentence.
```suggestion
* Previously, when you used the Agent-based Installer to install a cluster on a node that had an incorrect date, the cluster installation failed. With this release, the live ISO time synchronization is patched in the Agent-based Installer, preventing such installation issues. The patch fixes the date issue and configures the `/etc/chrony.conf` file with the list of additional Network Time Protocol (NTP) servers. You can set any of these additional NTP servers in the `agent-config.yaml`. (link:https://issues.redhat.com/browse/OCPBUGS-45464[*OCPBUGS-45464*])
```
